corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/collections.js

```diff
@@ -64,8 +64,10 @@ hqDefine("cloudcare/js/formplayer/menus/collections", function () {
             }
             if (response.selections) {
                 urlObject.setSelections(response.selections);
-                sessionStorage.removeItem('selectedValues');
                 updateUrl = true;
+                var currentSelectedValues = sessionStorage.selectedValues !== undefined && response.multiSelect ?
```

Comment:
This change seems fine, given the previous code, but it does make me curious: when is `selections` provided or not provided? I'd have thought it was provided on all `navigate_menu` responses, but that would include changing pages.

Reply:
ah yes I'm running into some QA bugs: [here](https://dimagi-dev.atlassian.net/browse/QA-4410) and [here](https://dimagi-dev.atlassian.net/browse/QA-4412) so not quite sure when to clear those instances and keep the pagination, but still investigating